<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PolyLineToolInterface</class>
 <widget class="QWidget" name="PolyLineToolInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>213</width>
    <height>158</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>213</width>
    <height>158</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>213</width>
    <height>158</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>213</width>
    <height>158</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>207</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="activeHLayout" stretch="1,2">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="activeLabel">
      <property name="text">
       <string>Target:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="activeComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="resetPLButton">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>36</y>
     <width>201</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset Polyline Tool</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="insidePLGroupBox">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>62</y>
     <width>203</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Inside Polyline</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>22</y>
      <width>203</width>
      <height>69</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="insideVLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QPushButton" name="insideFillButton">
       <property name="toolTip">
        <string>Flip across the axial plain</string>
       </property>
       <property name="text">
        <string>Fill</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="insideEraseButton">
       <property name="toolTip">
        <string>Flip across the coronal plain</string>
       </property>
       <property name="text">
        <string>Erase</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
